zerver/migrations/0023_auto_20160724_1657.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import bitfield.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0022_subscription_pin_to_top'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='stream',
+            name='default_permissions',
+            field=bitfield.models.BitField([b'can_read', b'can_write', b'can_moderate', b'can_add_remove_users'], default=1),
```

Comment:
Setting `default` to 1 is confusing because it's not immediately clear which element in the list that's referring to (do we read the list from the left or the right?). Better to replace `default=1` with `default=[b'can_read']`

`django-bitfield` supports a variety of ways to pass in `default`: https://github.com/disqus/django-bitfield/blob/master/bitfield/models.py#L122